addons/default/visiosoft/advs-module/src/AdvsModuleServiceProvider.php

```diff
@@ -217,6 +218,10 @@ class AdvsModuleServiceProvider extends AddonServiceProvider
             'as' => 'visiosoft.module.advs::user.configrations.create',
             'uses' => 'Visiosoft\AdvsModule\Http\Controller\OptionConfigurationController@create',
         ],
+
+        'advs/configuration/ajax/create' => 'Visiosoft\AdvsModule\Http\Controller\OptionConfigurationController@ajaxCreate',
+        'advs/configuration/ajax/delete' => 'Visiosoft\AdvsModule\Http\Controller\OptionConfigurationController@ajaxDelete',
```

Comment:
Anyone can delete anyone's config! You need to add an auth middleware and check if the ad belongs to the authenticated user